Kickstarter-iOS/Views/Controllers/SelectCurrencyViewController.swift

```diff
@@ -0,0 +1,79 @@
+import Library
+import Prelude
+import Prelude_UIKit
+import UIKit
+
+final class SelectCurrencyViewController: UIViewController, MessageBannerViewControllerPresenting {
+  private let viewModel: SelectCurrencyViewModelType = SelectCurrencyViewModel()
+
+  internal var messageBannerViewController: MessageBannerViewController?
+  private var saveButtonView: LoadingBarButtonItemView!
+
+  internal static func instantiate() -> SelectCurrencyViewController {
+    return SelectCurrencyViewController(nibName: nil, bundle: nil)
+  }
+
+  public func configure(with currency: Currency) {
+    self.viewModel.inputs.configure(with: currency)
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.view.addSubview(self.tableView)
+    self.tableView.constrainEdges(to: self.view)
+
+    self.viewModel.inputs.viewDidLoad()
+  }
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.tableView
+      |> settingsTableViewStyle
+      |> \.separatorStyle .~ .singleLine
+  }
+
+  // MARK: Subviews
+
+  private lazy var tableView: UITableView = {
+    let tableView = UITableView(frame: .zero, style: .plain)
```

Comment:
I know this is super readable but wonder if we can skip the need for declaring local variable by using the pipe

i.e.
```
return UITableView()
  |> translatesAutoresizingMaskIntoConstraints .~ false
  |> ...
```

Reply:
I think it's better to call the designated initializer and pass in an empty frame but sure we can look at piping it through keypaths.

Reply:
Designated initializer sounds fine...I was just being lazy re-typing everything :) the point was the pipe